### PR TITLE
Add highlighted resources section to instructor tab

### PR DESCRIPTION
**Pull request by RoyEJohnson**
Issue #983

### CODE REVIEW

File: src/app/helpers/controller/cms-mixin.js [hunk 1]
```diff
@@ -50,7 +46,7 @@ async function getUrlFor(slug) {
 
     // A little magic to handle the news slug
     if (slug === 'news') {
-        apiUrl = await newsPromise;
+        apiUrl = '/apps/cms/api/pages/openstax-news';
```
Comment (mwharrison) [inline src/app/helpers/controller/cms-mixin.js:49]: is it possible to use the API base from the settings file (I think you have this somewhere)? That way there is only one place to change the API path if it changes.. again.

File: src/app/pages/details/details.js [hunk 2]
```diff
@@ -64,6 +64,7 @@ export default class Details extends BaseClass {
 
         super.init();
         this.bookTitle = 'Loading';
+        console.info('Getting slug from', bookTitle);
```
Comment (mwharrison) [inline src/app/pages/details/details.js:67]: debug message?

File: src/app/helpers/controller/cms-mixin.js (hunk 1 above)
Comment (RoyEJohnson) [reply src/app/helpers/controller/cms-mixin.js]: Good catch

File: src/app/pages/details/details.js (hunk 2 above)
Comment (RoyEJohnson) [reply src/app/pages/details/details.js]: Got rid of a couple more.

**Pull request merged**